src/azul/plugins/metadata/hca/transform.py

```diff
@@ -82,6 +85,41 @@
 pass_thru_uuid4: PassThrough[api.UUID4] = PassThrough()
 
 
+class OrganismAge(FieldType[JSON, str]):
+
+    def to_index(self, value: JSON) -> str:
+        try:
+            value, unit = value['value'], value['unit']
+        except TypeError:  # Used for translating None
+            value, unit = [None, None]
+        except KeyError:
+            require(value.keys() == {'value', 'unit'},
```

Comment:
Don't we already know that one of the keys is missing? Why are we checking again?

Reply:
Yes. To trigger the requirement error.

Reply:
> To trigger the requirement error.

It would have been more professional conduct to not only answer my question literally but to also recognize that this is obviously an unnecessarily complicated means to that end.

Reply:
Noted, my bad.